index.html

```diff
@@ -256,7 +255,7 @@ <h1 class="font-sans text-xl font-semibold tracking-tight text-gray-100">
 				</div>
 			</div>
 		</div>
-		<div class="w-0 lg:w-1/2 h-0 lg:h-48 grid place-items-end invisible lg:visible">
+		<div class="w-0 lg:w-1/2 h-0 lg:h-48 grid place-items-end invisible lg:visible d-none lg:d-grid">
```

Comment:
I  hadn't noticed this. I appreciate the fix, however, you didn't need to implement your own classes. I'm using TailwindCSS which has classes for this: https://tailwindcss.com/docs/display

```suggestion
		<div class="w-0 lg:w-1/2 h-0 lg:h-48 grid place-items-end hidden lg:block">
```